### ISSUE TITLE
retry download fail

### ISSUE THREAD

**Issue by PikuZheng** [alexta69/metube#262]
get this error when trying to redownload a failed download
old version is perfect
```
ERROR:aiohttp.server:Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/aiohttp/web_protocol.py", line 433, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.8/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "app/main.py", line 108, in add
    status = await dqueue.add(url, quality, format, folder, custom_name_prefix)
  File "/app/app/ytdl.py", line 285, in add
    return await self.__add_entry(entry, quality, format, folder, custom_name_prefix, already)
  File "/app/app/ytdl.py", line 265, in __add_entry
    self.queue.put(Download(dldirectory, output, output_chapter, quality, format, self.config.YTDL_OPTIONS, dl))
  File "/app/app/ytdl.py", line 51, in __init__
    self.format = get_format(format, quality)
  File "/app/app/dl_formats.py", line 45, in get_format
    raise Exception(f"Unkown format {format}")
Exception: Unkown format test
```

**Comment by alexta69** [comment]
Which commit has broken this?

**Comment by PikuZheng** [comment]
I am expected to give more information on Sunday. sorry.

It is currently known that when re-downloading, the "Download Folder" becomes "Format" and the addition fails

**Comment by PikuZheng** [comment]
If the "Download Folder" is not default, this error occurs when the retry button is clicked after the download fails.
I did not find the reason. Sorry.

**Comment by alexta69** [comment]
Fixed by #310
